doc/adr/0009-container-image-tags.md

```diff
@@ -0,0 +1,55 @@
+# 9. Convention for tagging of container images
+
+Date: 2020-03-13
+
+## Status
+
+Proposed
+
+## Context
+
+Libero's products are assembled from product specific, general libero and external container images.
+
+Integration testing and reproducible deploys require uniquely identifiable image tags.
+
+
+## Decision
+
+### Tagging flow
+
+- each container image of a Libero container shall be tagged with the git-sha of the build
+- compose files used for integration test shall use immutable image tags
+- the compose files will track the component images and external images with the help of PR generating bots (e.g. dependabot, renovate)
+- upon passed integration tests the used libero component images get tagged with the git-sha of the integration/umbrella repo
+- semver tagging of Libero component images is triggered by tagging of the umbrelle repo
+
+### Tag Structure
+
+Component build passes tests and gets pushed to registry:
+
+- liberoadmin/reviewer-client:full_git_hash
+
+Integration tests in umbrella repo succeed:
+
+- liberoadmin/reviewer-client:reviewer-full_git_hash
```

Comment:
Think i'm missing some context around this? Maybe we could talk about this in the sync meeting this week?

Reply:
Is there a "Glossary of Terms" - struggling with the term "umbrella repo"